draft-ietf-quic-recovery.md

```diff
@@ -403,18 +403,25 @@ until the handshake is confirmed.
 
 smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.
 
-When there are no samples for a network path, and on the first RTT sample for
-the network path:
+An endpoint needs to bootstrap the RTT estimator during connection establishment
+and when the estimator is reset during connection migration (Section 9.4 of
+{{QUIC-TRANSPORT}}).
+
+When there are no RTT samples for a new network path, and on the first
+subsequent RTT sample for the new network path, smoothed_rtt and rttvar are set
+as follows:
 
 ~~~
 smoothed_rtt = rtt_sample
 rttvar = rtt_sample / 2
 ~~~
 
-Before any RTT samples are available, the initial RTT is used as rtt_sample.  On
-the first RTT sample for the network path, that sample is used as rtt_sample.
-This ensures that the first measurement erases the history of any persisted or
-default values.
+Before any RTT samples are available for a new path or when the estimator is
```

Comment:
How about moving these two paragraph above the paragraph starting with "When there are no RTT samples..." and reworking the text a bit as a result?

Reply:
I actually did exactly that, and realized that it's awkward to use `rtt_sample` in that structure, since it's not yet been used. But yeah, this still isn't great. I'll work on it a bit.

Reply:
Ok, done.